enterprise_access/apps/subsidy_request/tasks.py

```diff
@@ -27,3 +34,72 @@ def update_course_title_for_subsidy_request_task(subsidy_type, subsidy_request_u
 
     # Use bulk_update so we don't trigger save() again
     subsidy_model.bulk_update([subsidy_request], ['course_title'])
+
+
+@shared_task(base=LoggedTaskWithRetry)
+def send_admins_email_with_new_requests_task(enterprise_customer_uuid):
```

Comment:
Long what do you mean by
> pull both license & coupon codes in request agnostically
?

Reply:
this task currently sends both new license & coupon code requests in the same email, but we need a distinction between them since there is a button in the email that leads the admin to the correct page.

Reply:
fixed in latest PR (dynamically creates the url based on the subsidy type associated with the subsidy config)

thanks for this callout btw